exercises/rail-fence-cipher/.meta/src/reference/java/RailFenceCipher.java

```diff
@@ -0,0 +1,64 @@
+public class RailFenceCipher {
+
+    private static int key;
```

Comment:
I don't think this or any of the methods should be static. Either all the methods are static and you pass in the key to each method, or the methods are all instance methods and you pass in the key to the constructor. Since we prefer instance methods according to our [policies doc](https://github.com/exercism/java/blob/master/POLICIES.md#prefer-instance-methods) I would suggest the latter :)